### PR TITLE
chore(blog): add first newsletters as blog post

### PR DESCRIPTION
**Pull request by argshook**
This PR is adds first 3 newsletters as blog posts

### CODE REVIEW

**Review by argshook — COMMENTED** [review]
@amyshwang this PR is example of how content from mailchimp would look in blog post. Please check.

I'll put here one more newsletter soon.

File: blog/2020-12-08-newsletter-1.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+title: Newsletter No. 1
+author: QuestDB
```
[the remark this answers is not in the document]
Comment (amyshwang) [reply blog/2020-12-08-newsletter-1.md:3]: Its shown as a hyperlink but not linking to anything...

File: blog/2020-12-08-newsletter-1.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+description:
+  We've been working hard making QuestDB more reliable and extending the
+  functionality in response to what our community expects from a
+  high-performance time series database.
+tags: [newsletter]
+image: /img/blog/2020-12-08/banner.png
```
Comment (amyshwang) [inline blog/2020-12-08-newsletter-1.md:9]: Should we use the later banner for newsletter? I can provide the original file of:
image

File: blog/2020-12-08-newsletter-1.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+import Banner from "@theme/Banner"
+
+<Banner
+  alt="Image of QuestDB logo"
+  src="/img/blog/2020-12-08/banner.png"
+  width={300}
+  height={76}
+/>
+
+# December Updates
```
Comment (amyshwang) [inline blog/2020-12-08-newsletter-1.md:21]:
```suggestion

```

File: blog/2020-12-08-newsletter-1.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
+---
+title: Newsletter No. 1
```
Comment (amyshwang) [inline blog/2020-12-08-newsletter-1.md:2]:
```suggestion
title: Newsletter Dec 2020
```

File: blog/2020-12-08-newsletter-1.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+author: QuestDB
+description:
+  We've been working hard making QuestDB more reliable and extending the
+  functionality in response to what our community expects from a
+  high-performance time series database.
+tags: [newsletter]
+image: /img/blog/2020-12-08/banner.png
+---
```
Comment (amyshwang) [inline blog/2020-12-08-newsletter-1.md:10]: As discussed: 

```suggestion
slug: newsletter-dec-2020
---
```

File: blog/2020-12-08-newsletter-1.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+# December Updates
+
+2020 is quickly drawing to an end and we'd like to wrap up the month on a high
+note with you. We've been working hard making QuestDB more reliable and
+extending the functionality in response to what our community expects from a
+high-performance time series database.
+
+We have more exciting features, functionality, and content in recent weeks and
+we hope you enjoy reading about it.
+
```
Comment (amyshwang) [inline blog/2020-12-08-newsletter-1.md:30]:
```suggestion
<!--truncate-->
```

**Review by amyshwang — COMMENTED** [review]
I made comments on the high-level structure. I have also checked the links.

File: blog/2020-12-08-newsletter-1.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (argshook) [reply blog/2020-12-08-newsletter-1.md:3]: we always have `author_url` so there's only one way to render `author` - as a link.

i can add `author_url: https://questdb.io/about` maybe?

[the remark this answers is not in the document]
Comment (argshook) [reply blog/2020-12-08-newsletter-1.md:3]: i saw you added `author_url` and `author_image_url`, both of which point to github. Non ideal to point to github for images, but that's what we do now for all blog posts, so let's keep for consistency.

it's a good idea, so i did the same for other newsletters as well.

File: blog/2020-12-08-newsletter-1.md (hunk 2 above)
Comment (argshook) [reply blog/2020-12-08-newsletter-1.md]: yep, good idea. I see you already updated, nice!

**Review by amyshwang — APPROVED** [review]

**Pull request merged**